apps/web/src/components/activity/ExecutionDetailTrigger.tsx

```diff
@@ -11,34 +11,25 @@ const TriggerTitle = styled(Text)`
   padding-bottom: 20px;
 `;
 
-const buildTrigger = (identifier, subscriber, payload): INotificationTrigger => {
-  const subscriberVariables = [{ name: 'subscriberId', value: subscriber._id }];
-
-  if (subscriber.email) {
-    subscriberVariables.push({
-      name: 'email',
-      value: subscriber.email,
-    });
-  }
-
-  if (subscriber.phone) {
-    subscriberVariables.push({
-      name: 'phone',
-      value: subscriber.phone,
-    });
-  }
+const buildTrigger = (identifier, subscriberVariables, payload): INotificationTrigger => {
+  subscriberVariables = JSON.parse(subscriberVariables || '{}');
```

Comment:
What's the reason behind doing the variable global?

Reply:
global?

Reply:
Didn't see it in the arguments, sorry.
Not big fan of modifying them through the memory reference through. Good to go anyway.